htmd/molecule/voxeldescriptors.py

```diff
@@ -148,33 +148,52 @@ def _findDonors(mol, bonds):
     return donors
 
 
-def _getSigmas(mol):
-    """ Returns the VDW radii of each atom in the molecule
-
+def _getRadii(mol):
+    """ Gets vdW radius for each elem in mol.element. Source VMD.
+    
     Parameters
     ----------
     mol :
-        A Molecule object
+        A Molecule object. Needs to be read from Autodock 4 .pdbqt format
 
     Returns
     -------
-    sigmas : np.ndarray
-        The radius of each atom in Molecule
+    radii : np.ndarray
+        vdW radius for each element in mol.
```

Comment:
Couldn't a better source be found? I don't even understand what elements like A or D mean